docs/features/java.rst

```diff
@@ -183,18 +219,22 @@ be necessary to allocate the buffer and free it later.
 The default capacity of pool buffers can be modified with an Open MPI
 MCA parameter::
 
-  shell$ mpirun --mca mpi_java_eager SIZE ...
+  $ mpirun --mca ompi_mpi_java_eager SIZE ...
 
-Where ``SIZE`` is the number of bytes, or kilobytes if it ends with 'k',
-or megabytes if it ends with 'm'.
+The value of ``SIZE`` can be:
+
+* ``N``: An integer number of bytes
+* ``Nk``: An integer number (suffixed with ``k``) of kilobytes
+* ``Nm``: An integer number (suffixed with ``m``) of megabytes
 
 An alternative is to use "direct buffers" provided by standard classes
-available in the Java SDK such as ``ByteBuffer``. For convenience we
-provide a few static methods ``new[Type]Buffer`` in the ``MPI`` class to
-create direct buffers for a number of basic datatypes. Elements of the
-direct buffer can be accessed with methods ``put()`` and ``get()``, and
-the number of elements in the buffer can be obtained with the method
-``capacity()``. This example illustrates its use:
+available in the Java SDK such as ``ByteBuffer``. For convenience,
+Open MPI provides a few static methods ``new[Type]Buffer`` in the
```

Comment:
could this be ``new[Type]Buffer(LENGTH)`` ?

Reply:
Errr... I don't know enough Java to know.  😄  That line of docs is from the original authors.

Reply:
It looks like an MPI classism - it's not really a java thing:

https://github.com/open-mpi/ompi/pull/10256/files#diff-5523b1ba17ab0028b18c91c93b443aa9e3d2117556d1c9f627fbc57c4ff67db8L204

The typical java `new` for an array would be ``int[] buf = new buf[SIZE]``.

Reply:
Would be fine for a subsequent commit. 😄